src/components/pages/taxonomy/list/table/taxon-modal/observation/use-observation-count.tsx

```diff
@@ -0,0 +1,35 @@
+import { axGetListData } from "@services/observation.service";
+import { useEffect } from "react";
+import { useImmer } from "use-immer";
+
+export default function useObsCount(taxonId) {
```

Comment:
this arguement can be renamed as taxon so while calling `axGetListData` you can just use it as is